python/BUILD

```diff
@@ -21,10 +21,13 @@
 
 load("@vaticle_dependencies//builder/swig:python.bzl", "py_native_lib_rename")
 load("@vaticle_dependencies//tool/checkstyle:rules.bzl", "checkstyle_test")
+load("//python/tool/sphinx:rules.bzl", "sphinx_docs")
+load("//tool/docs:rules.bzl", "html_docs_parser")
 
 load("//python:rules.bzl", "native_driver_versioned")
 load("//python:python_versions.bzl", "python_versions")
 
+load("@vaticle_dependencies_tool_docs//:requirements.bzl", docs_requirement = "requirement")
```

Comment:
Conceptually you shouldn't need to load python requirements here? this is more of a WORKSPACE thing I think